test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenHandler.ValidateTokenAsyncTests.Lifetime.cs

```diff
@@ -0,0 +1,219 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.TestUtils;
+using Microsoft.IdentityModel.Tokens;
+using Xunit;
+
+namespace Microsoft.IdentityModel.JsonWebTokens.Tests
+{
+    public partial class JsonWebTokenHandlerValidateTokenAsyncTests
```

Comment:
why partial? also I can't get these tests to show up in the VS test runner.

Reply:
got it work after re-starting

Reply:
Explained on the other PR, but the main idea is that we split all the regression tests across files for readability and ease to follow. We are separating them by aspect of the validation: lifetime, audience, issuer, replay, actor, signature, token type, encryption, and by using the partial class we can have all the test methods show under the single class in the text explorer.